index.js

```diff
@@ -16,11 +16,11 @@ module.exports = function (content) {
       children = result.attributes.children,
       componentProps = renderProps(props);
 
-    let imports = '';
+    let imports = 'import React from "react";';
```

Comment:
Ok, you are right, I was just trying it out, wanted to be sure the import would not create additional instances of React, thanks for contributing!